web/modules/custom/bnf/bnf_client/src/Plugin/QueueWorker/SubscriptionNewContent.php

```diff
@@ -113,4 +126,20 @@ public function processItem($data): void {
     }
   }
 
+  /**
+   * Check if a node with the given UUID is locally claimed.
+   */
+  protected function isLocallyClaimed(string $uuid): bool {
+    $nodes = $this->nodeStorage->loadByProperties(['uuid' => $uuid]);
+    $node = reset($nodes);
+
+    if (!$node instanceof NodeInterface) {
+      return FALSE;
+    }
+
+    $state = (int) $node->get(BnfStateEnum::FIELD_NAME)->value;
```

Comment:
And the AI is missing the point here. The field module provides the underlying enum, that's kinda the point of the `enum_field` module. See how it's done in https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/1d0677008b5d8661727e2cbf57c3e457ca04d5e4/web/modules/custom/bnf/bnf_client/bnf_client.module#L108

This implementation couples this code to the enum type. If it didn't (as the above example), the states could change from `int` to `string` without having to fix this.